src/store/actions.js

```diff
@@ -1,6 +1,6 @@
 import axios from 'axios'
 
-const BASE_API_PATH = '/api/v3'
+const BASE_API_PATH = 'http://localhost:8081/api/v3'
```

Comment:
You will probably get some CORS errors with this. You should configure the Proxy in the vite configuration

Reply:
done